### DISCUSSION TITLE
Official Playlists from Vevo and Labels

### DISCUSSION THREAD

**Discussion by StigtriX** [nukeop/nuclear#1043] [Ideas]
Would there be a way to make a tab on the dashboard with official playlists?
What I mean is a section which pulls playlists from vevo and/or the different labels. Or maybe there are members with musical expertise wou could make curated playlists? 😄 

The dashboard is already more functional than the mess Spotify currently is. Good job! 👍

**Comment by nukeop** [comment]
We're working on a service for sharing playlists: https://github.com/NuclearPlayer/nuclear-web-services

**Reply by nukeop** [reply]
This will also allow subscribing to curated playlists at some point

**Reply by StigtriX** [reply]
Nice! I did not see that post. Thank you for letting me know 🙂
